src/pretix/control/templates/pretixcontrol/checkin/index.html

```diff
@@ -0,0 +1,74 @@
+{% extends "pretixcontrol/event/base.html" %}
+{% load i18n %}
+{% load eventurl %}
+{% load urlreplace %}
+{% block title %}{% trans "Check-Ins" %}{% endblock %}
+{% block content %}
+    <h1>{% trans "Check-ins" %}</h1>
```

Comment:
Here it's 'Check-ins', two lines above you used a capital I. I think we go for lowercase I consistently?